senza/aws.py

```diff
@@ -24,6 +25,24 @@ def get_security_group(region: str, sg_name: str):
             raise
 
 
+def encrypt(region: str, KeyId: str, Plaintext: str, encode=False):
+    kms = boto3.client('kms', region)
+    encrypted = kms.encrypt(KeyId=KeyId, Plaintext=Plaintext)['CiphertextBlob']
+    if encode:
+        return base64.b64encode(encrypted).decode('utf-8')
+
+    return encrypted
+
+
+def list_kms_keys(region: str, describe=False):
```

Comment:
this function is called only with describe=True, you can remove this parameter altogether (or make True a default)...